view-creator-framework/src/test/resources/sample-view-generation-spec.conf

```diff
@@ -1,18 +1,12 @@
 tool.class = org.hypertrace.core.viewcreator.pinot.PinotTableCreationTool
 service.name = "sample-view"
+
 view = {
   name = myView
   output.schema.class = org.hypertrace.core.viewcreator.test.api.TestView
   output.schema.url = "host:port/myView"
 }
 
-kafka = {
-  brokerAddress = "localhost:9092"
-  topicName = test-view-events
-  partitions = 8
-  replicationFactor = 3
-}
-
 pinot = {
```

Comment:
we don't need a separate block for offline config ?

Reply:
its optional. added it anyway and added tests for it.